integration_test/workflow_tests/test_packages/ingest/test_no_granules.py

```diff
@@ -43,20 +46,23 @@ def test_no_granules_passes(self):
             expected_output = {
                 "payload": {"granules": [], "copied_to_glacier": []},
                 "meta": {
-                    "collection": {"name": collection_name, "version": collection_version},
+                    "collection": {
+                        "name": collection_name,
+                        "version": collection_version,
+                    },
                     "provider": {"id": provider_id, "name": provider_name},
                 },
                 "cumulus_meta": {"execution_name": execution_id},
                 "task_config": {
-                    "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+                    "orcaExcludedFileRegEx": "{$.meta.collection.meta.orca.orcaExcludedFileRegEx}",
```

Comment:
Shouldn't this be `orcaExcludedFileExtensions` not `orcaExcludedFileRegEx`

Reply:
oops forgot